modules/services/analytics/src/main/java/au/com/shiftyjelly/pocketcasts/analytics/di/TrackerModule.kt

```diff
@@ -35,4 +37,10 @@ object TrackerModule {
     internal fun provideFirebaseAnalytics(@ApplicationContext context: Context): FirebaseAnalyticsWrapper {
         return FirebaseAnalyticsWrapper(FirebaseAnalytics.getInstance(context))
     }
+
+    @Provides
+    @Singleton
+    fun provideExperimentProvider(firebaseRemoteConfig: FirebaseRemoteConfig): ExperimentProvider {
+        return ExperimentProvider(firebaseRemoteConfig)
+    }
```

Comment:
You don't need to do this when there is `@Inject` on `ExperimentPorivder`'s constructor.

Reply:
Thanks for pointing this out, I have fixed this up.